pkg/kubelet/network/hairpin/hairpin.go

```diff
@@ -37,7 +37,7 @@ const (
 )
 
 var (
-	ethtoolOutputRegex = regexp.MustCompile("peer_ifindex: (\\d+)")
+	ipLinkOutputRegex = regexp.MustCompile("eth0@if(\\d+)")
```

Comment:
@thockin I think we should keep it in v1.4, but after the release we might remove it if it's unneeded.